src/main/steps/respondent/task-list/tasklist.ts

```diff
@@ -107,8 +108,8 @@ const getTheApplicationSection = (taskListItems, userCase: CaseWithId, userIdamI
     itemList.push({
       id: 'check_the_application',
       text: taskListItems.check_the_application,
```

Comment:
Why are we removing it, can you please explain

Reply:
Actually this should be part of response task list . In ticket its mentioned for respondent task list .. so I migrated it to task list response.